startup_scripts/20_custom_fields.py

```diff
@@ -36,8 +36,11 @@ def get_class_for_class_path(class_path):
         if cf_details.get('description', 0):
           custom_field.description = cf_details['description']
 
-        if cf_details.get('filterable', 0):
-          custom_field.is_filterables = cf_details['filterable']
+        # If no filter_logic is specified then it will default to 'Loose'
+        if cf_details.get('filter_logic', 0):
+          for choice_id, choice_text in CF_FILTER_CHOICES:
```

Comment:
If I get [the python docs](https://docs.python.org/2/library/stdtypes.html#typesseq) right you should be able to do `CF_FILTER_CHOICES.index(choice_text.lower())`.

Reply:
No this is not possible as the values in the variable `CF_FILTER_CHOICES` is a tuple containing several tuple objects. The extract of the inner tuple object is some syntax sugar that python provides.

```
>>> from extras.constants import CF_FILTER_CHOICES
>>> CF_FILTER_CHOICES.index('Disabled')           
Traceback (most recent call last):                
  File "<console>", line 1, in <module>           
ValueError: tuple.index(x): x not in tuple        
>>> CF_FILTER_CHOICES.index('disabled')           
Traceback (most recent call last):                
  File "<console>", line 1, in <module>           
ValueError: tuple.index(x): x not in tuple        
```

Reply:
Alright, thank you for trying it.